contrib/debian/rules

```diff
@@ -213,6 +213,9 @@ override_dh_fixperms:
 	# local-listeners
 	chmod 4750 $(TOP)/usr/libexec/netdata/plugins.d/local-listeners
 
+	# systemd-journal
+	chmod 4750 $(TOP)/usr/libexec/netdata/plugins.d/systemd-journal.plugin
+
```

Comment:
Correction to my earlier statement, can we drop this for the moment? It’s failing right now due to the plugin not actually getting built (due to missing deps which will need to be handled in a separate PR).